Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml

```diff
@@ -4,7 +4,7 @@ commonfields:
 name: ArcusTeam
 display: ArcusTeam
 category: Data Enrichment & Threat Intelligence
-description: 'The ArcusTeam API allows the user to inspect connected devices'' attack surface. By feeding device identifiers and the software it runs: DeviceTotal will return a map of the device’s attack surface. DeviceTotal was built from the ground up in order to provide complete visibility into connected devices and mitigate 3rd party risk. DeviceTotal can continuously identify & predict such that the connected device security posture is being assessed, prioritized and mitigated effectively.'
+description: "The ArcusTeam API allows the user to inspect connected devices' attack surface. By feeding device identifiers and the software it runs: DeviceTotal will return a map of the device\u2019s attack surface. DeviceTotal was built from the ground up in order to provide complete visibility into connected devices and mitigate 3rd party risk. DeviceTotal can continuously identify & predict such that the connected device security posture is being assessed, prioritized and mitigated effectively."
```

Comment:
this is weird, @anas-yousef device\u2019s

Reply:
Fixed